packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -111,16 +106,16 @@ export class AzureClient {
         const module = { fluidExport: runtimeFactory };
         const codeLoader = { load: async () => module };
         const urlResolver = new AzureUrlResolver(
-            this.connectionConfig.tenantId,
-            this.connectionConfig.orderer,
-            this.connectionConfig.storage,
-            this.connectionConfig.tokenProvider,
+            this.config.connectionConfig.tenantId,
```

Comment:
Spreading would allow us to access the `config` directly instead of having to go through `connectionConfig`. Although perhaps the difference is intended to differentiate between optional/mandatory parameters?

Reply:
Change naming from config to props which I think makes this answer more clear.